pulpcore/cli/common/generic.py

```diff
@@ -42,6 +42,21 @@
 _ = translation.gettext
 
 
+class IncompatibleContext(click.ClickException):
+    exit_code = 0
+
+    def show(self, file: Optional[IO[str]] = None) -> None:
+        """
+        Format the message into file or STDERR.
+        Overwritten from base class to print "IncompatibleContext: ".
+        """
+        if file is None:
+            file = click.get_text_stream("stderr")
+        click.echo(
+            _("IncompatibleContext: {message}").format(message=self.format_message()), file=file
+        )
```

Comment:
I think less is more:
```suggestion
    pass
```

First this exception is signifying a user error so "exit 1" is appropriate. Second let's not copy code from click if we do not really need to.

Reply:
I decided to make this change to better inform the user about the possibly wrong use of context. I can update `exit_code`. But, I am in favour of leaving the error message formatting untouched.